force-app/main/default/lwc/attendance/attendance.js

```diff
@@ -31,24 +35,28 @@ import SUBMIT_LABEL from "@salesforce/label/c.Submit";
 import ATTENDANCE_LABEL from "@salesforce/label/c.Attendance";
 import LOADING_LABEL from "@salesforce/label/c.Loading";
 import SUCCESS_LABEL from "@salesforce/label/c.Success";
-import SUCCESS_MESSAGE_LABEL from "@salesforce/label/c.Save_Attendance_Success";
-import SUCCESS_NO_UPDATES_LABEL from "@salesforce/label/c.Save_Attendance_No_Updates";
+import SUCCESS_MESSAGE_LABEL from "@salesforce/label/c.Save_Service_Delivery_Success";
+import SUCCESS_NO_UPDATES_LABEL from "@salesforce/label/c.Save_Service_Delivery_No_Updates";
 import UPDATE_LABEL from "@salesforce/label/c.Update";
 import SAVE_LABEL from "@salesforce/label/c.Save";
 import CANCEL_LABEL from "@salesforce/label/c.Cancel";
 import QUANTITY_LABEL from "@salesforce/label/c.Quantity";
+import PRINT_LABEL from "@salesforce/label/c.Print";
+import PRINTABLE_VIEWABLE_LABEL from "@salesforce/label/c.Printable_View";
 import NO_PARTICIPANTS_HEADER_LABEL from "@salesforce/label/c.No_Participants_Header";
 import NO_PARTICIPANTS_MESSAGE_LABEL from "@salesforce/label/c.No_Participants_Message";
 import NO_PERMISSIONS_MESSAGE_LABEL from "@salesforce/label/c.No_Permission_Message";
+import pmmFolder from "@salesforce/resourceUrl/pmm";
 
 const FIELD_SET_NAME = "Attendance_Service_Deliveries";
 const SHORT_DATA_TYPES = ["DOUBLE", "INTEGER", "BOOLEAN"];
 const LONG_DATA_TYPES = ["TEXTAREA", "PICKLIST", "REFERENCE"];
 const ID = "Id";
 const COMPLETE = "Complete";
 const PENDING = "Pending";
-
-export default class Attendance extends LightningElement {
+const PAGE_NAVIGATION_TYPE = "standard__navItemPage";
```

Comment:
Not sure this really matters, so feel free to ignore me. What about ITEM_PAGE_NAVIGATION_TYPE ? My think is that when we are reading down below we can easily identify what is stored.